apps/pie-docs/src/patterns/cookie-banner/code/props.json

```diff
@@ -0,0 +1,56 @@
+{
+    "headings": ["Prop", "Options", "Description", "Default"],
+    "rows": [
+        [
+            "hasPrimaryActionsOnly",
+            {
+                "type": "code",
+                "item": ["true", "false"]
+            },
+            "When true, sets the variant to \"primary\" for the button which accepts necessary cookies only.",
+            {
+                "type": "code",
+                "item": ["\"false\""]
```

Comment:
I think this should be a boolean?

Reply:
I don't get this one :) 
we are not capturing the types in the props table, I do think maybe we should though. Is that what you are referring to?

Reply:
Hopefully this is clearer. In the options column they appear as booleans but in the default column it looks like a string
![2024-04-30 08_57_29](https://github.com/justeattakeaway/pie/assets/26894168/59c39fa3-c317-4e17-a4a9-c08fd096d9da)

Reply:
thanks! excellent catch! 🙌